src/Silverback.Integration/Messaging/Inbound/ErrorHandling/MoveMessageErrorPolicy.cs

```diff
@@ -140,6 +140,7 @@ public override bool CanHandle(ConsumerPipelineContext context, Exception except
             private async Task PublishToNewEndpointAsync(IRawInboundEnvelope envelope, Exception exception)
             {
                 envelope.Headers.AddOrReplace(DefaultMessageHeaders.SourceEndpoint, envelope.ActualEndpointName);
+                envelope.Headers.AddOrReplace(DefaultMessageHeaders.SourceConsumer, envelope.Endpoint.GetUniqueConsumerGroupName());
```

Comment:
Are you sure this is what we want? Don't we want to use the plain and simple `GroupId`?

Reply:
Right, thanks, it should be now fixed.